lib/Doctrine/DBAL/Driver/SQLSrv/SQLSrvStatement.php

```diff
@@ -20,7 +20,8 @@
 namespace Doctrine\DBAL\Driver\SQLSrv;
 
 use Doctrine\DBAL\Driver\StatementIterator;
-use PDO;
```

Comment:
This is bug - there is still unqualified `PDO::FETCH_ORI_NEXT` reference.

Reply:
Looks like the only place like this. Please see https://github.com/doctrine/dbal/pull/3024.